README.md

```diff
@@ -42,6 +38,17 @@ Documentation | Package
 
 Read the [contributing guidelines](CONTRIBUTING.md) file if you are interested in contributing.
 
+## Getting Started
+
+  1. Install [Node.js and npm](https://nodejs.org/en) <!-- TODO: use a dev container or NixOS maybe? -->
+      - Use the correct [engine version](package.json): [n](https://www.npmjs.com/package/n), [nvm](https://github.com/nvm-sh/nvm), or other
+  2. Use `pnpm` by default `corepack enable pnpm`
+  3. Install dependencies project `npm install`
+  4. Check that gravity is still working - `npm test` - before making any changes
```

Comment:
This will likely happen as part of a `prepush` git hook in the future, this job also runs in all PRs so it should be caught before a merge takes place if there is a failed test or anything.

Reply:
I was thinking of this as specific to initial setup as a check that everything is installed correctly.

Reply:
Fair enough in that case!